Scripts/Python/plasma/Plasma.py

```diff
@@ -1270,6 +1289,27 @@ def stop(self):
         """Stops the animation"""
         pass
 
+class ptAsyncTask(Awaitable):
+    """An asynchronously completed task"""
+    def __init__(self, aw: Optional[Awaitable[T_co]] = None):
+        ...
+
+    def __await__(self) -> Iterator[Optional[T_co]]:
+        ...
+
```

Comment:
With the above fix, you shouldn't need to manually hint `__await__`, as it will be inherited (with the correct types) from `Awaitable`:

```suggestion
```